modules/gapi/src/backends/cpu/gcpuvideo.cpp

```diff
@@ -107,13 +107,111 @@ GAPI_OCV_KERNEL_ST(GCPUBackgroundSubtractor,
     }
 };
 
+cv::gapi::video::KalmanParams::KalmanParams(int dp, int mp, int cp, int tp)
```

Comment:
I think this constructor should be removed since majority of the matrices will be changed from default valued ones by user. We can add this constructor in the future if needed

Reply:
Done.